src/app/news/news-list.component.ts

```diff
@@ -82,6 +87,25 @@ export class NewsListComponent implements OnChanges {
       this.dialogsFormService.closeDialogsForm();
       this.dialogsLoadingService.stop();
     });
+    this.sendNewsNotifications(oldNews);
+  }
+
+  sendNewsNotifications(oldNews) {
+    const userId = this.userService.get()._id;
+    const link = '/community';
```

Comment:
**NewsListComponent** is not specific to the community news, so the link could be for a team or enterprise.  You'll need to construct this from `this.viewableBy` and `this.viewableId`.